contents/founders/first-1000-users.md

```diff
@@ -0,0 +1,231 @@
+
+And we kept talking to all our existing users about what they did and didn’t like about the product, and the problems they wanted to solve.
+
+Over the course of a few weeks we gathered all this intel and mapped out what users loved, how to scale it, and anti-goals we should avoid. This lead us to focus on shipping more features engineers wanted, such as feature flags, that they also wanted to self-host.
+
+Here’s an example of what this looks like for PostHog today. Notice that the anti-goals are all standard things that most companies follow, which is exactly why bigger companies often become unpopular at scale.
+
+![goals](https://res.cloudinary.com/dmukukwp6/image/upload/goals_and_anti_goals_324c9044a8.png)
+
+At this stage, I also started experimented with content as a marketing channel. I'm not good at coding, so figured I could help out this way.
+
+I knew from these conversations that developers wouldn't appreciate being sold to, but I guessed they would find following our journey interesting. So, I wrote about what my co-founder and I were up to – how we pivoted lots of times, or what it was like moving to San Francisco for YC.
+
+This got us a couple more hits on the front page of Hacker News, which proved it worked, so we kept doing it. That’s why, four years and 60,000 customers later, I'm writing this blog post, as the CEO, at half midnight.
+
+### Getting our first sales
+
+Our successful HackerNews launch also gave us the confidence to start thinking about pricing.
+
+I did two things to get our very first sales:
+
+1. I talked to users that were already using our product and in our Slack community about our paid offering and gauging interest.
+
+2. I created a pricing page on our website, listing our paid extra features and embedding my calendar into it, gradually making this self-serve over time. When someone booked a call, I'd see which extra features they really cared about, and if they wanted to buy.
+
+It was easier to get sales from totally new users via our pricing page than from the existing community, though both approaches worked. I suspect this is because some existing users were using us because it was free.
+
+![history](https://res.cloudinary.com/dmukukwp6/image/upload/first_six_months_0a5368c875.png)
+
+In early pricing discussions, I negotiated ad-hoc with customers, going from "will anyone pay anything?" to a consistently used internal doc, which I iterated until it felt consistently right. 
+
+We had two goals at this stage:
+
+1. To figure out if people would pay anything for what we’d built.
+2. Create a pricing model technical users like ourselves would appreciate, which meant it had to be self-serve and totally transparent. 
+
+After a few weeks of trial and error, we had a public pricing table and started letting people put their card details in. Every time we made it more self-serve, we got a higher conversion rate.
+
+### Launching PostHog Cloud
```

Comment:
I think you could probably combine one of the paragraphs in this section, but I'm not sure which.